django/core/management/commands/shell.py

```diff
@@ -114,6 +116,11 @@ def handle(self, **options):
             exec(options['command'])
             return
 
+        if select.select([sys.stdin], [], [], 0.0)[0]:
```

Comment:
0.0 -> 0

Also what about windows support? `select` module docs say at the top "Note that on Windows, it only works for sockets;" and under `select` func, "a straightforward interface to the Unix select() system call.". [This SO post](https://stackoverflow.com/questions/12499523/using-sys-stdin-in-select-select-on-windows) implies it's not possible to use `select` either.